pyproject.toml

```diff
@@ -16,11 +16,11 @@ builtins = [
 	"npgettext",
 ]
 
-logger-objects = ["logHandler.log"]
-
 include = [
 	"*.py",
 	"*.pyw",
+	"sconstruct",
+	"*sconscript",
```

Comment:
Ugh, I now notice that this gets really noisy. We should either tell Ruff about scons internals like `Return`, `Import`, etc. or revert this, I think.

Reply:
This seems fixable with 

```
[tool.ruff.lint.per-file-ignores]
# sconscripts contains many inbuilt functions not recognised by the lint,
# so ignore F821.
"sconstruct" = ["F821"]
"*sconscript" = ["F821"]
```